### PR TITLE
Make it possible to print results in the console rather than sending them to a distant endpoint.

### PR DESCRIPTION
**Pull request by bnjbvr**
With this pull request, there is a new option: --submitter (aka -s), which can take two values: 'remote' or 'print'. Default is 'remote'. When the submitter is set to remote, results will be sent to the AWFY php service, as it's done currently. When the submitter is set to print, results will be printed once we're done, which allows to run AWFY locally more easily.

### CODE REVIEW

File: driver/dostuff.py [hunk 1]
```diff
@@ -40,11 +43,19 @@
                ]
 Engines, NumUpdated = builders.build(KnownEngines, not options.noupdate, options.force)
 
+Submitter = None
+if options.submitter == 'remote':
+    Submitter = submitter.RemoteSubmitter
+elif options.submitter == 'print':
+    Submitter = submitter.PrintSubmitter
+else:
+    raise Exception('unknown submitter!')
```
Comment (h4writer) [inline driver/dostuff.py]: Can you create getSubmitter(name) which has this logic.
Like:
https://github.com/bnjbvr/arewefastyet/blob/submitter/browser-driver/benchmarks_local.py#L75

File: driver/dostuff.py [hunk 2]
```diff
@@ -40,11 +43,19 @@
...
+
 # No updates. Report to server and wait 60 seconds, before moving on
 if NumUpdated == 0 and not options.force:
     for slave in slaves.init():
-        submit = submitter.Submitter(slave)
-        submit.Awake();
+        remotecontroller = RemoteController(slave)
+        remotecontroller.Awake();
```
Comment (h4writer) [inline driver/dostuff.py]: Can you use Submitter for this too?
And test if submitter is 'remote' before sending Awake.

**Comment by h4writer** [comment]
Awesome! Good to split "Controller" from "Submitter". Can you address the two issues raised above?

File: driver/dostuff.py (hunk 2 above)
Comment (bnjbvr) [inline driver/dostuff.py]: I don't understand. Do you want to have the Awake method in RemoteSubmitter, is that it? It is in RemoteController right now. I am ok with moving it, but that would make RemoteController a dummy class with only the RequestedRevs.

Do you prefer that I move back these two functions to RemoteSubmitter and entirely delete the RemoteController?

Comment (h4writer) [inline driver/dostuff.py]: If I understand correctly RemoteSubmitter extends RemoteController right? So 'submitter' can also be used to call "Awake()". So if you add a test: "if options.submitter == 'remote'" everything should work right? Or was that not the intention?

**Pull request merged**